### PR TITLE
Cleanup duplicate "Overview" lines in GoDoc

### PR DESCRIPTION
**Pull request by bwerthmann**
GoDoc appends a package's doc item from all files in the package.

Example:
https://pkg.go.dev/github.com/nats-io/nats-surveyor@v0.1.1/surveyor#pkg-overview

Signed-off-by: Ben Werthmann <ben@synadia.com>

### CODE REVIEW

**Comment by bwerthmann** [comment]
Lint is failing in CI:
https://github.com/nats-io/nats-surveyor/runs/4436497974?check_suite_focus=true#step:5:39

File: surveyor/surveyor.go [hunk 1]
```diff
@@ -11,7 +11,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package surveyor is used to garner data from a NATS deployment for Prometheus
```
Comment (ripienaar) [inline surveyor/surveyor.go:14]: prefer we leave it here and not add an empty doc file personally, though this pattern is common enough that it's probably fine.  So LGTM.

Comment (bwerthmann) [reply surveyor/surveyor.go]: Makes sense to me. Pushed a commit for this.

**Comment by ripienaar** [comment]
> Lint is failing in CI: https://github.com/nats-io/nats-surveyor/runs/4436497974?check_suite_focus=true#step:5:39

Update CI, before calling `go vet` call `go mod tidy`. It's a hack, like `go mod` so par for the course.

**Pull request merged**